model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2021, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2020-2021 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Remove "(c)" to avoid copyright wrap around.

Reply:
Updated.